src/publish/publish.js

```diff
@@ -281,6 +291,30 @@ async function publishChart(request, h) {
     };
 }
 
+async function loadVendorLocale(vendor, locale) {
+    const basePath = path.resolve(
+        __dirname,
+        '../../node_modules/@datawrapper/locales/locales/',
+        vendor
+    );
+    const culture = locale.replace('_', '-').toLowerCase();
+    const tryFiles = [`${culture}.js`];
+    if (culture.length > 2) {
+        // also try just language as fallback
+        tryFiles.push(`${culture.substr(0, 2)}.js`);
+    }
+    for (let i = 0; i < tryFiles.length; i++) {
+        const file = path.join(basePath, tryFiles[i]);
+        try {
+            return await fs.readFile(file, 'utf-8');
```

Comment:
```suggestion
            return fs.readFile(file, 'utf-8');
```
No need to `await` here. You can even remove the `async` from the function declaration.